components/grid/grid.mdx

```diff
@@ -0,0 +1,63 @@
+---
+name: Grid
+---
+
+import { Playground, PropsTable } from 'docz'
+import Row from '../row'
+import Col from './styles'
```

Comment:
shouldn't be `../col`?

Reply:
I made a custom style for Col in this case to add a border, just to illustrate the sizes.